po/es.po

```diff
@@ -1662,48 +1655,22 @@ msgstr "no se encontró ningún canal RSS"
 
 #: rss/rssparserfactory.cpp:31
 msgid "unsupported feed format"
-msgstr "formato de fuente no soportado"
+msgstr "formato de fuente no admitido"
 
-#, fuzzy
 msgid "%s: %s: invalid loglevel value"
-msgstr "`%s' es un tipo de diálogo inválido"
+msgstr "%s: %s: nivel de registro incorrecto"
 
-#, fuzzy
-msgid ""
-"Fatal error: couldn't determine home directory!\n"
+msgid "Fatal error: couldn't determine home directory!\n"
 "Please set the HOME environment variable or add a valid user for UID %u!"
-msgstr ""
-"¡Por favor define la variable de entorno HOME, o añade un usuario válido "
-"para UID %u!"
+msgstr "Error fatal: no se pudo determinar el directorio del usuario.\n"
+"Por favor, define la variable de entorno HOME o añade un usuario válido "
+"para UID %u."
```

Comment:
Ditto about punctuation (exclamation mark replaced by period).

Reply:
Withdrawn, [see explanation in a comment below](https://github.com/newsboat/newsboat/pull/874#issuecomment-612408710).